torch/csrc/api/src/nn/init.cpp

```diff
@@ -95,6 +92,8 @@ Tensor dirac_(Tensor tensor) {
       case 5: // Volumetric convolution
         tensor[d][d][sizes[2] / 2][sizes[3] / 2][sizes[4] / 2] = 1;
         break;
+      default:
```

Comment:
I think this is intentional so that we can detect if we have an unhandled case? Wouldn't some unreachable macro be better here?

Reply:
The default  case can be removed since there is a check for dimension before.